BLSQ/iaso
# WC2-327 entities infos

entities infos was displayed twice


<img width="753" alt="Screenshot 2024-02-02 at 16 28 08" src="https://github.com/BLSQ/iaso/assets/12494624/71aa8f6d-8bdc-4a09-9c75-01a6c70e5550">


Related JIRA tickets : WC2-327

## Self proofreading checklist




## Changes

display string or number, not both

## How to test

Open a beneficiary detail and check that data is not displayed as string and as a number

## Print screen / video

<img width="854" alt="Screenshot 2024-02-02 at 16 27 50" src="https://github.com/BLSQ/iaso/assets/12494624/a5a52cb6-62ff-4cd9-8336-804e3873927d">
